api-client/src/errors/ResponseError.js

```diff
@@ -33,10 +33,12 @@ class ResponseError extends Error {
   }
 
   createMessage ({ requestOptions, url, endpointName, response }: ResponseErrorParamsType): string {
+    // This is a react-native FormData
```

Comment:
Does this still work for web?

Reply:
No it actually does not. Any ideas?

Reply:
Without changing this feedback does not work for native? Or what was the reason for this?

Reply:
Found it. See https://github.com/taylesworth/react-native-mock/blob/06bb395c8f2b7228716db4bb29330d10a60b65bb/src/react-native.js#L127 . Maybe there is a way to keep the original form data. But as everything else seems to be unaffected it may be enough to provide a functionality for iterating the form data object depending on the platform.